docs/lib/project-setup/fragments/flutter/null-safety/null-safety.md

```diff
@@ -0,0 +1,18 @@
+**Amplify Flutter and Null Safety**
+
+
+The Amplify Flutter library supports [Dart null safety](https://dart.dev/null-safety) beginning with version 0.2.0.  This means that:
+
+* Applications utilizing [sound null safety](https://dart.dev/null-safety) must use Amplify Flutter version 0.2.0 or greater.
+
+* Applications utilizing [unsound null safety](https://dart.dev/null-safety/unsound-null-safety) can use versions of Amplify Flutter above or below 0.2.0, but some application changes may be necessary to migrate to 0.2.0.
+
```

Comment:
Do you think it would be easier to make a table of what works and doesn’t.  This way we can have less text and make it faster for customers to understand what works with what.

Reply:
Personally I prefer it as-is, but we can change it if you feel strongly.

Reply:
+1 for the table idea

Reply:
What are you think for a table? Are you thinking of something like the table below? I personally don't see a ton of value in a table since its not much info. I also sort of prefer the way it is, but I think either way gets the info across.


|                                	| amplify-flutter 0.1.6 and below   	| amplify-flutter 0.2.0 and above  	|
|-------------------	|---------------------------------	|---------------------------------	|
| Non Null Safe App 	| Supported                                    	| Supported                                    	|
| Null Safe App     	    | Not Supported                             	| Supported                                    	|

Reply:
I missed the third bullet initially. I guess you could add a third row to that table for the third bullet. I don't think you can have a null safe app that isn't running w/flutter 2+ so I don't think we really have to consider that.

|                                	                     | amplify-flutter 0.1.6 and below   	| amplify-flutter 0.2.0 and above  	|
|-------------------------------	|---------------------------------	|---------------------------------	|
| Null Safe App     	                         | Not Supported                             	| Supported                                    	|
| Non Null Safe App w/ flutter v2  | Supported                                    	| Supported                                    	|
| Non Null Safe App w/ flutter v1 	| Supported                                    	| Not Supported                                    	|